EasyPost.Tests.Net35/EasyPost.Tests.Net35.csproj

```diff
@@ -0,0 +1,111 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <TargetFramework>net452</TargetFramework>
+
+        <IsPackable>false</IsPackable>
```

Comment:
Will this keep the tests from getting tossed in with the rest of the files then? Do we do this on our base test files?

Reply:
What are you referring to here?

Reply:
Basically, this line is going to exclude tests from the packaged nuget library right?

Reply:
Correct.